recipe/build.sh

```diff
@@ -1,5 +1,10 @@
 export DISABLE_AUTOBREW=1
 
+# https://github.com/conda-forge/r-changeforest-feedstock/issues/28
+if [[ $target_platform == osx-64 ]]; then
+  export TARGET="aarch64-apple-darwin"
```

Comment:
This does not look correct. This is setting`aarch64-apple-darwin` on **osx-64** platform. The fact that this passed would indicate that it had no effect.

Reply:
What's the value of `target_platform` for osx-arm64?

Reply:
I would guess **osx-arm64**?